From 6f8edf9f04928435cac811b3e1bfe333197ea066 Mon Sep 17 00:00:00 2001
From: ninez <johnstonljordan@gmail.com>
Date: Thu, 6 Jan 2022 14:26:19 -0500
Subject: [PATCH 1/3] revert MR 2173: no new gsettings schema

 Gnome-Shell-git depends on new gsettings schemas, but as i am running 41.x on
 Archlinux, for now revert this patch.

Signed-off-by: ninez <johnstonljordan@gmail.com>
---
 .gitlab-ci.yml | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml
index b2eff112c..4477a942a 100644
--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -33,16 +33,11 @@ variables:
     FDO_DISTRIBUTION_EXEC: |
       dnf install -y 'dnf-command(builddep)' &&
 
-      dnf builddep -y gsettings-desktop-schemas --setopt=install_weak_deps=False &&
       dnf builddep -y mutter --setopt=install_weak_deps=False &&
       dnf builddep -y gnome-shell --setopt=install_weak_deps=False &&
       dnf builddep -y libinput --setopt=install_weak_deps=False &&
       dnf builddep -y wayland-protocols --setopt=install_weak_deps=False &&
 
-      ./.gitlab-ci/install-meson-project.sh \
-        https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas.git \
-        master . 79dd34316ffa44221bcd1b6227aebe1b4b2643d8 &&
-
       ./.gitlab-ci/install-meson-project.sh \
         https://gitlab.freedesktop.org/wayland/wayland-protocols.git \
         1.24 . &&
-- 
2.34.1

